snooty.toml

```diff
@@ -10,3 +10,8 @@ version = 4.0
 package-name-org = "docs-ecosystem"
 pgp-version = "{+version+}"
 csharp-docs-versioned = "https://mongodb.github.io/mongo-csharp-driver/2.14"
+
+java-version = 4.4
+java-full-version = "{+java-version+}.0"
+java-package-name-org = "mongodb-org"
```

Comment:
What is this used for?

Reply:
This was a copy and paste of the [snooty constants from docs java](https://github.com/mongodb/docs-java/blob/master/snooty.toml#L11-L14). In hindsight, I should not have included unused constants. Removing `java-full-version` and `java-package-name-org`.